tests/rptest/remote_scripts/control/rw_alive.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+if [ ! -f /opt/remote/var/rw.pid ]; then
```

Comment:
I afraid that the alternative is worse (to embed bash into python as string and to pass to the server over ssh). Bash already is a tricky language when we add additional friction we only increase the level of attention we need to modify that area. 

Having them as separate files simplifies the debugging: we don't need to the python harness test the bash scripts. Imagine if there is a typo in the python bash generator, it's harder to catch on review, there is no syntax highlight for bash in python and we would need to add something to inspect the ephemeral ad-hoc commands.

Reply:
Perhaps a compromise would be to templatize the bash helpers?  That way we could have a template for "run this java program" start/stop/alive scripts, and write them out for each service from the template, rather than having multiple near-identical scripts for different services.